playkit/src/main/java/com/kaltura/playkit/drm/WidevineModularAdapter.java

```diff
@@ -315,9 +316,9 @@ private MediaDrmSession openSessionWithKeys(FrameworkMediaDrm mediaDrm, String k
         return session;
     }
 
-    private byte[] executeKeyRequest(String licenseUrl, ExoMediaDrm.KeyRequest keyRequest) throws Exception {
+    private byte[] executeKeyRequest(String licenseUrl, ExoMediaDrm.KeyRequest keyRequest, String licenseServerUrl) throws Exception {
         HttpMediaDrmCallback httpMediaDrmCallback = new HttpMediaDrmCallback(licenseUrl, buildDataSourceFactory());
-        return httpMediaDrmCallback.executeKeyRequest(MediaSupport.WIDEVINE_UUID, keyRequest);
+        return httpMediaDrmCallback.executeKeyRequest(MediaSupport.WIDEVINE_UUID, keyRequest, licenseServerUrl);
```

Comment:
I think this should be:
    `return httpMediaDrmCallback.executeKeyRequest(MediaSupport.WIDEVINE_UUID, keyRequest, licenseUrl);`
(see above).